### ISSUE TITLE
Telegram channel not found

### ISSUE THREAD

**Issue by kdltr** [mvdicarlo/postybirb-plus#281]
The telegram sender doesn’t work if no message has been sent in the channel for a while, saying that it’s not found.

If I send a message in the channel and restart PostyBirb, it works.

![Screenshot_2023-11-11_10-46-52](https://github.com/mvdicarlo/postybirb-plus/assets/19929894/8a424091-fc14-444e-856c-318a5a6cc17c)

**Comment by leaftail1880** [comment]
This happens due to limit latest of telegram chats postybirb can request (currently 200)

After request, postybirb filters them to exclude those channels where you cannot send images/messages and usually its around 10/20 chats. So when there is a lot messages in other chats postybirb cannot find older chat

**Comment by leaftail1880** [comment]
@kdltr Still actual?

**Comment by kdltr** [comment]
Still happens yes. But the message for it is much better  now ^^

**Comment by leaftail1880** [comment]
@kdltr try closing postybitb, then posting something on channel, opening postybirb, selecting channel and then deleting message on the channel.

**Comment by kdltr** [comment]
Yep, that’s what I’ve been doing as a workaround this whole time, no worries :)
